_posts/2016-11-10-tell-us-your-story-at-cfgmgmtcamp-ghent.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Tell us your story at CfgMgmtCamp Ghent / FOSDEM 2017 / DevConf
+date: 2016-11-10 00:01:54
+author: Foreman
+tags:
+- foreman
+- community
+- fosdem
+- cfgmgmtcamp
+- cfp
+excerpt: |
+  The Foreman Community wants to hear from YOU at our upcoming conferences.
+  Read on for details!
+---
+
+In just over 2 months, the yearly chaos of the Foreman convention season will
+be upon us. FOSDEM! CfgMgmtCamp! DevConf! Busy, busy, busy, and we want you to
+be a part of it! Come tell us your story.
+
+For these events to be successful, we need lots of good Foreman talks - and we
+want to hear your Foreman story. These conferences aren't just for developers
+to show off the latest cool feature (although that's always nice), but rather
+serve as an opportunity for the community to share it's knowledge, swap
+stories, and learn from each other. To do that, we need a mix of talks, from
+dev previews (as mentioned) to success stories, lessons from the front line,
+plugins used to solve a problem and so on.
```

Comment:
Updated to mention meeting face-to-face and to raise the emphasis on non-dev submissions.